life/modernized/02-life-of-gilbert-latey.adoc

```diff
@@ -568,6 +568,9 @@ and according to his normal manner, visited them in the jails,
 and endeavoured that nothing should be lacking for their support, or,
 according to his power, to procure their enlargement.
 
+[.embedded-content-document.testimony]
```

Comment:
I wasn't sure what to label this. Seemed like more of a testimony than anything else to me!

Reply:
And a very encouraging read given the current pandemic, although this pales in comparison to that. :)